### ISSUE TITLE
Accordion Accessibility Tab updates and DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon-website#572]
This issue is related to #491
The following updates are needed to the Accordion Accessibility Tab:
**1. Remove the first paragraph:** 
"The accordion React Carbon component has been tested against the latest W3C Web Content Accessibility Guidelines (WCAG) 2.1 Level A and AA success criteria and violations have been identified as high priority issues. This document will be updated when these accessibility issues are resolved."

**Note:** This paragraph is part of the Non-compliant template and should not be added to the completed documents. The non-compliant template is used when the document can not be completed due to open accessibility issues that block completion.

**2. Typically a different font is used for navigation keys.** Is there a Carbon font that can be used? If there is, the following navigation keys should be updated in the first paragraph under how it works:
Tab 
Shift-Tab
Enter
Space 

**3. Under the Heading "Accessibility Testing" add the following paragraph:**
Accessibility issues are tracked in the [Carbon Component GitHub repository](https://github.com/carbon-design-system/carbon/milestone/57).

**4. Update the  Automated Test Environment to the latest versions and include the DAP ruleset and Carbon React Version:**

macOS Mojave version 10.14.6 with VoiceOver
Chrome Version Version 77.0.3865.90
Dynamic Assessment Plugin Version 1.8.0.0 - IBM Accessibility WCAG 2.1 Sept. 2019 Ruleset
Carbon React Version 7.7.1

**5. Update Automated Test Environment > Results because new DAP violations were reported.  Change to: "DAP Test - Violations"** 

For macOS screen reader tests > Environment:
Change the first instance to:   
macOS Mojave version 10.14.6 with VoiceOver
Chrome Version Version 77.0.3865.90
Carbon React Version 7.7.1

Change the second instance to:
macOS Mojave Version 10.14.6 with VoiceOver
Safari Version 13.0.2
Carbon React Version 7.7.1

**6. The following DAP violations are also found. (Let me know if they need to be reported on a separate issue (see screenshot below:**
<img width="1318" alt="Screen Shot 2019-11-11 at 2 09 42 PM" src="https://user-images.githubusercontent.com/21676914/68617456-f6287300-048c-11ea-9c48-128e7d010ac3.png">
